lm_eval/tasks/score/README.md

```diff
@@ -0,0 +1,81 @@
+# SCORE: Systematic COnsistency and Robustness Evaluation for Large Language Models
+
+## License
+
+Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+## References
+[1] Wang, Yubo, et al. "Mmlu-pro: A more robust and challenging multi-task language understanding benchmark." arXiv preprint arXiv:2406.01574 (2024).
+
+[2] Zhong, Wanjun, et al. "Agieval: A human-centric benchmark for evaluating foundation models." arXiv preprint arXiv:2304.06364 (2023).
+
+[3] Hendrycks et al. "Measuring Mathematical Problem Solving With the MATH Dataset." arXic:2103.03874 (2021).
+
+## Citation
+```bib
+[Citation placeholder]
+```
+
+## Groups
+
+- `score_robustness_mmlu_pro`: two 0-shot robutstness tasks on MMLU-PRO dataset
+
+- `score_robustness_agieval`: two 0-shot robutstness tasks on the AGIEVAL datasets multiple choice questions subsets:  `'agieval-sat-math'`, `'agieval-lsat-lr'`, `'agieval-lsat-rc'`, `'agieval-logiqa-en'`, `'agieval-aqua-rat'`, `'agieval-sat-en'`, `'agieval-lsat-ar'` 
+
+- `score_robustness_math`: one 0-shot robutstness tasks on Hendryk's MATH dataset
+
+## Tasks
+
+Both `score_robustness_mmlu_pro` and `score_robustness_agieval` contain the following 2 tasks:
+
+* Option order robustness: 
+`score_option_order_robustness_mmlu_pro`, 
+`score_option_order_robustness_agieval`
+
+* Prompt robustness: 
+`score_prompt_robustness_mmlu_pro`, 
+`score_prompt_robustness_agieval`,
+
+Whereas math contains only
+* Prompt robustness:
+`score_prompt_robustness_math`
+
+
+### Option order robustness
+
+Measures the model's robustness to the placment of the correct answer in the options list by swapping the correct answer with all the other possible options.
+
+### Prompt robustness
+
+Measures the model's robustness to 10 different prompts. list of the prompts can be found in the `./prompt_templates.json` file under the key `prompt_robustness`.
+
+
+## Notes
+
+- All The tasks are designed for **Instruct** models for which we recommend to pass "`--apply_chat_template`" flag.
+
```

Comment:
how about adding a section about metrics [acc/cr...+reference to the paper that introduces CR] here to make it clear that all predictions/evals are aggregated to the CR? we can then use this in the leaderboard docs